.github/actions/post-metric/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Post Datadog Metric
+description: GitHub Action to post a metric to DD.
+
+inputs:
+  datadog-token:
+    description: 'token for datadog'
+    required: true
+  metric:
+    description: 'metric name for datadog'
+    required: true
+  metric-value:
+    description: 'metric value for datadog'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Post Given Metric to Datadog using API.
+      shell: bash
+      id: post_metric
+      run: |
+        export NOW="$(date +%s)"
```

Comment:
Let's use this NOW for now, but our next step should be to change the `yarn bench` script to record its own timestamp, and extract it from the JSON file.